.github/workflows/test_pr.yml

```diff
@@ -144,8 +145,7 @@ jobs:
 
       - name: Test
         run: |
-          python -m pytest -k test_egrabber --color=yes --cov-report=xml --cov=acquire --maxfail=5 --log-cli-level=0
-
+          python -m pytest -k test_egrabber --color=yes --cov-report=xml --cov=acquire --maxfail=5 --log-cli-level=0 --doctest-glob="*.pyi" --doctest-glob="*.md"
```

Comment:
Instead of repeatedly specifying `--doctest-glob`, it's probably better to put these in the pytest.ini options `addopts` in the `pyproject.toml`. Unless there's a reason to only run the doctests for some jobs?